### PR TITLE
fix panic: duplicate metrics collector registration attempted

### PR DESCRIPTION
**Pull request by yancl**
## Which problem is this PR solving?
fix duplicate metrics collector registration attempted panic

## Short description of the changes
use different names for counter and gauge metrics collector.

### CODE REVIEW

File: cmd/ingester/app/consumer/offset/manager.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func NewManager(minOffset int64, markOffset MarkOffset, partition int32, factory
 	return &Manager{
 		markOffsetFunction:  markOffset,
 		close:               make(chan struct{}),
-		offsetCommitCount:   factory.Counter("offset-commit", map[string]string{"partition": strconv.Itoa(int(partition))}),
+		offsetCommitCount:   factory.Counter("offset-commit-count", map[string]string{"partition": strconv.Itoa(int(partition))}),
 		lastCommittedOffset: factory.Gauge("offset-commit", map[string]string{"partition": strconv.Itoa(int(partition))}),
```
Comment (yurishkuro) [inline cmd/ingester/app/consumer/offset/manager.go]: Counter-proposal:
* counter: offset-commits-total ("-total" is recommended by Prometheus docs)
* gauge: last-committed-offset

@vprithvi ^^^

**Comment by yurishkuro** [comment]
change lgtm. Please refer to CONTRIBUTING.md for instructions on getting the PR in shape to be merged (CI build, and DCO check).

**Comment by yurishkuro** [comment]
And thanks for contributing!

**Comment by yancl** [comment]
Thanks, I have made some changes, PTAL:)

**Comment by vprithvi** [comment]
Thanks for your contribution. LGTM, but could you document what exactly is panicking in the PR?  (I'm assuming it's the prom reporter)

**Comment by black-adder** [comment]
Prometheus doesn't allow initialization of the same name,tag combination.

**Pull request merged**